api/types.proto

```diff
@@ -277,6 +277,41 @@ message UpdateConfig {
 
 	// Amount of time between updates.
 	Duration delay = 2 [(gogoproto.nullable) = false];
+
+	enum OnFailure {
+		PAUSE = 0;
```

Comment:
I discussed this with @aluzzardi earlier and we decided on "pause". I think the issue with "abort" is it could make people think that the update is rolled back. "Pause" describes the state where some tasks are updated but the rest are not.